packages/core/src/invoke.js

```diff
@@ -3,7 +3,6 @@
 /** @author John Hann */
 
 export default function invoke (f, args) {
-  /* eslint complexity: [2,7] */
```

Comment:
Hmmm, I wonder if when we switched to standard we lost our `complexity` rule settings.  I think we previously had it intentionally set quite low, like 3 or 4--hence the explicit complexity comments.  If that's the case, then I think the right thing may be to re-enable our low complexity setting (if that's possible with standard), and keep or update all the complexity comments.

I'd like to hear from @TylorS, @Frikki, and @davidchase to see how they feel about the complexity rule as well.

Reply:
I feel the complexity rule is worthwhile when you have less experienced developers that could introduce unnecessary complexity. However, they would quickly resort to disabling the rule with a comment. Of course, this can be a good thing because attention will be brought to it during code reviews, but maybe we would spot the complexity anyway.

Reply:
iirc standard itself doesnt have any feelings towards complexity, however we can extend standard and add specific rules if we wanted. i think keeping complexity and at reasonable level would be good for us and any other contributor especially less experienced as @Frikki mentions 😄

Reply:
I'm definitely in favor of re-introducing the complexity rule--seems like we lost it in the switch to standard.  We [had it set to 3](https://github.com/mostjs/eslint-config-most/blob/master/index.js#L11).  I like a very low setting as a default, so that we have to explicitly raise it or disable it in spots that are more complex.

What do you think?

Reply:
I am game.

Reply:
hooya